CHANGES.rst

```diff
@@ -154,6 +154,9 @@ tweakreg
   another. It is an analog of the ``tweakback`` task in the
   ``drizzlepac``. [#7573, #7591]
 
+- Added the 'GAIADR3' catalog to the available options for alignment; 
+  this has been enabled as the default option [#7217].
```

Comment:
```suggestion
  this has been enabled as the default option [#7611].
```
The number referenced in the change log entry is the PR number, not the JP ticket number.